validation/src/main/scala/hmda/validation/rules/lar/quality/Q024.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.quality
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object Q024 extends EditCheck[LoanApplicationRegister] {
+  val config = ConfigFactory.load()
+  val min_income = config.getInt("hmda.validation.quality.Q024.min_income_for_high_loan")
+
+  override def name: String = "Q024"
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    val income = lar.applicant.income
+
+    when((lar.actionTakenType is equalTo(1)) and (income is numeric)) {
+      when(lar.loan.amount is greaterThanOrEqual(income.toInt * 5)) {
```

Comment:
I wanted to make sure `toInt` could never run on a non-numeric income. (And I wanted that to be clear to the reader).